lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -145,12 +145,13 @@
   when: full_provision
 
 - name: Install OpenAirInterface (OAI) dependencies
+  retries: 5
```

Comment:
Do we really need that? seems like a hack. maybe file a task to fix this no?

Reply:
not really added -- was present in original task, which was copied as a template for the new per-distribution tasks